opennms-correlation/drools-correlation-engine/src/main/java/org/opennms/netmgt/correlation/drools/DroolsCorrelationEngine.java

```diff
@@ -85,17 +92,31 @@
     private String m_eventProcessingMode;
     private boolean m_isStreaming = false;
     private final Meter m_eventsMeter;
+    private MetricRegistry m_metricRegistry;
     private Boolean m_persistState;
+    private Resource m_configPath;
+    private ApplicationContext m_configContext;
     
-    public DroolsCorrelationEngine(final String name, final MetricRegistry metricRegistry) {
+    public DroolsCorrelationEngine(final String name, final MetricRegistry metricRegistry, final Resource configPath, final ApplicationContext configContext) {
         this.m_name = name;
+        this.m_configPath = configPath;
```

Comment:
I know, I was just following the convention used on hundreds of files on our code base, but I can remove it needed.